extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -95,6 +95,7 @@ export let InteractiveReactData;
  *    canShowPreviousPageHelp: boolean,
  *    canShowSharingUis: boolean,
  *    canShowSystemLayerButtons: boolean,
+ *    customUI: JSONObject,
```

Comment:
Let's make it specific to the additional controls. If we add extra UI we might want to add extra StateProperties, as it will be intended for different components it'll be great to only listen for what components need.